scripts/build/gen_syscalls.py

```diff
@@ -396,7 +397,7 @@ def analyze_fn(match_group, fn, userspace_only):
         if args == "void":
             args = []
         else:
-            args = [typename_split(a.strip()) for a in args.split(",")]
+            args = [typename_split(a) for a in args.split(",")]
```

Comment:
Why do you need to remove `.split()` here? This strips out newline too.

Reply:
This moves the `strip` when processing just args to being done inside `typename_split`. So `strip` runs for func type+name and also args type+name. The `strip` is now more centralized and shared.